mock/src/constants.rs

```diff
@@ -39,6 +41,15 @@ pub const DEFAULT_ACCOUNT_CODE: &str = "
     export.basic_eoa::auth_tx_rpo_falcon512
 ";
 
+// Default authentication code
+pub const DEFAULT_AUTH_CODE: &str = "
```

Comment:
nit: I would change this to `DEFAULT_AUTH_SCRIPT`.

Reply:
Should this be called `DEFAULT_ACCOUNT_SCRIPT` then?

```rust
// Default account code
pub const DEFAULT_ACCOUNT_CODE: &str = "
    use.miden::contracts::wallets::basic->basic_wallet
    use.miden::contracts::auth::basic->basic_eoa

    export.basic_wallet::receive_asset
    export.basic_wallet::send_asset
    export.basic_eoa::auth_tx_rpo_falcon512
";
```

Reply:
We actually use the term "script" when we talk about notes and term "code" when we talk about accounts. The distinction is that "script" is an executable program while account code is a collection of procedures.

So, keeping this as `DEFAULT_ACCOUNT_CODE` is fine.